Makefile

```diff
@@ -33,4 +33,4 @@ drgn_tools/_version.py:
 .PHONY: rsync
 rsync: drgn_tools/_version.py
 	@if [ -z "$(TARGET)" ]; then echo "error: TARGET unspecified. Either set it in config.mk, or use\nmake TARGET=hostname rsync"; exit 1; fi
-	rsync -avz --exclude "__pycache__" --exclude ".git" --exclude ".mypy_cache" ./drgn_tools $(TARGET):drgn_tools/
+	rsync -avz --exclude "__pycache__" --exclude ".git" --exclude ".mypy_cache" ./drgn_tools $(TARGET)
```

Comment:
To ensure that this doesn't break any existing users who may already have TARGET set and expect it to go to `TARGET:drgn_tools/`, would you please go ahead and add the following code into the Makefile? Directly beneath the `-include config.mk` directive would be best.
```makefile
# The TARGET variable could be either a hostname or a hostname:path. Previously
# the makefile expected just a hostname, and it always placed the resulting
# directory at $TARGET:drgn_tools/. But it's nice to have flexibility to change
# the target directory, so we only add that default path if there's not already
# a path specified.
ifneq (,$(TARGET))
ifeq (,$(findstring :,$(TARGET)))
override TARGET := $(TARGET):drgn_tools/
else
endif
endif
```